test/functional/plugin/tutor_spec.lua

```diff
@@ -0,0 +1,67 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear = helpers.clear
+local command = helpers.command
+local feed = helpers.feed
+local is_os = helpers.is_os
+local Screen = require('test.functional.ui.screen')
+
+describe(':Tutor', function()
+  clear({ args = { '-u', 'NORC' } })
+  command('set cmdheight=0')
+  command('Tutor')
+
+  it('should apply transformation', function()
+    local test_table --[[@as table]]
+    if is_os('win') then
+      test_table = {
```

Comment:
a ternary would avoid the need for the explicit type
```suggestion
    local test_table = is_os('win') and {
    } or {
    }
```